<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ForecastWidget</class>
 <widget class="QWidget" name="ForecastWidget">
  <property name="windowTitle">
   <string>Прогноз</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelDate">
       <property name="text">
        <string>Дата прогноза:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDateEdit" name="editDate">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
       <property name="date">
        <date>2025-01-01</date>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelIncome">
       <property name="text">
        <string>Ожидаемые доходы:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="editExpectedIncome">
       <property name="maximum">
        <double>1000000.00</double>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelExpense">
       <property name="text">
        <string>Ожидаемые расходы:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="editExpectedExpense">
       <property name="maximum">
        <double>1000000.00</double>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelComment">
       <property name="text">
        <string>Комментарий:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="editComment"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="btnAddForecast">
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableForecasts"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>

